telegram/_telegramobject.py

```diff
@@ -327,6 +341,34 @@ def to_dict(self, recursive: bool = True) -> JSONDict:
             :obj:`dict`
         """
         out = self._get_attrs(recursive=recursive)
+
+        # Now we should convert TGObjects to dicts inside objects such as sequences, and convert
+        # datetimes to timestamps. This mostly eliminates the need for subclasses to override
+        # `to_dict`
+        for key, value in out.items():
+            if isinstance(value, Sequence) and not isinstance(value, str):
+                val = []  # empty list to append our converted values to
```

Comment:
```suggestion
                if len(value) == 0:
                    out[key] = value
                     continue
                     
                val = []  # empty list to append our converted values to
```

go, early returns, go! 🤡 That way you can also skip lines 365-366